### DISCUSSION TITLE
Order of repos within list

### DISCUSSION THREAD

**Discussion by ivanperez-keera** [community/community#49250] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Lists are a great way of keeping an eye on your own repos. It can help keep things grouped and organized when you don't want to create a separate organization for them.

To help us keep things even more organized, please let us sort both list **AND repos** _alphabetically_. Setting the order of repos by default to most 'recent activity' or 'higher number of stars' makes searching harder. Specifically, it improves the general case because what you are looking for is likely to be on top; _but search becomes O(n) instead of O(log n)_. Also, it makes it harder to use spatial memory, and it limits accessibility (some people are likely to miss details but can easily rely on spatial memory to locate things if they don't move much).

**Comment by 87968** [comment]
Un propietario de este repositorio ha limitado la capacidad de comentar de nuevos usuarios

**Reply by behindthebooks** [reply]
Yes it gets hard to keep looking for a list to add a repository. Even if lists are sorted alphabetically it will be easy to locate a list quickly. Thanks for this awesome feature 🙏

**Reply by hrada777** [reply]
> Un propietario de este repositorio ha limitado la capacidad de comentar de nuevos usuarios

Red

**Comment by GheeSlimoe** [comment]
Settings.

**Reply by vicitm44444** [reply]
Hello

**Comment by 0583231610** [comment]
...

**Reply by 0583231610** [reply]
0583231610

**Comment by Sirleid** [comment]
[]()

**Comment by Sirleid** [comment]
👍
